client/blocks/taxonomy-manager/style.scss

```diff
@@ -13,4 +13,36 @@
 
 .taxonomy-manager__list-item {
 	background-color: $gray-light;
-}
\ No newline at end of file
+}
+
+.taxonomy-manager__list-item-card {
+	cursor: pointer;
+}
+
+.taxonomy-manager__list-item-toggle {
+	color: $gray;
+	cursor: pointer;
+	font-size: 24px;
+	margin: 17px 28px;
+	position: absolute;
+		right: 0;
+		top: 0;
```

Comment:
The extra tab for positioning styles is part of our CSS guidelines ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#positioning)).